konduit-serving-core/src/main/java/ai/konduit/serving/metrics/RegressionMetrics.java

```diff
@@ -145,11 +148,11 @@ public MetricsConfig config() {
     @Override
     public void updateMetrics(Object... args) {
         if(args instanceof Record[]) {
-            Record[] records = (Record[]) args[0];
+            Record[] records = (Record[]) args;
             incrementClassificationCounters(records);
         }
         else if(args instanceof INDArray[]) {
-            INDArray[] output = (INDArray[]) args[0];
+            INDArray[] output = (INDArray[]) args;
             incrementClassificationCounters(output);
 
```

Comment:
Maybe write it as you've done for the classification metrics:
```
        if(args[0] instanceof Record) {
            Record records = (Record) args[0];
            incrementClassificationCounters(new Record[]{records});
        }
        else if(args[0] instanceof Record[]) {
            Record[] records = (Record[]) args[0];
            incrementClassificationCounters(records);
        }
        else if(args[0] instanceof INDArray) {
            INDArray output = (INDArray) args[0];
            incrementClassificationCounters(new INDArray[] {output});
        }
        else if(args[0] instanceof INDArray[]) {
            INDArray[] output = (INDArray[]) args[0];
            incrementClassificationCounters(output);
        }
```

so that the records can be passed in as both `updateMetrics(new INDArray[] {...})` or `updateMetrics(INDArray, INDArray)`